.github/workflows/test-all.yml

```diff
@@ -272,11 +279,39 @@ jobs:
           python -m pip install -r tools/requirements.txt
 
       - name: "Generate test-status.json"
-        id: report
-        run: python tools/generate_test_status.py ${{ github.repository }} "$GITHUB_RUN_ID" "$GITHUB_SHA" $(git rev-parse --short "$GITHUB_SHA")
-
-      - name: "Output test-status"
         id: test-status
         run: |
-          TEST_STATUS=$(cat test-status.json)
-          echo "::set-output name=test-status::$TEST_STATUS"
+          ROOT='data/reports'
+          DIR_REL=$(python tools/generate_test_status.py ${{ github.repository }} "$GITHUB_RUN_ID" "$GITHUB_SHA" ${{ github.event.inputs.which-gap }})
+          DIR="${ROOT}/${DIR_REL}"
+          echo "::set-output name=dir::${DIR}"
+          echo "::set-output name=dir-rel::${DIR_REL}"
+
+      - name: "Download latest report"
+        id: download-latest-report
+        run: |
+          ROOT="data/reports"
+          DIR_SYM_REL=latest-${{ github.event.inputs.which-gap }}
+          DIR_SYM="${ROOT}/${DIR_SYM_REL}"
+          URL_SYM="https://raw.githubusercontent.com/${{ github.repository }}/${DIR_SYM}"
+          if curl --head --fail --silent "${URL_SYM}" 2>/dev/null; then
```

Comment:
What does this do? I could probably figure out by reading the curl man page / docs, but a brief comment would help... It's especially "mysterious" because here you use `curl` and then below `wget`... why?!

Reply:
Nevermind, I think `wget` has also a --head option, so I will try to get rid of `curl` here. Then there is no confusion why I switch tools.